.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Thank you for contributing to The Foreman, please read the following guide
+available at https://www.theforeman.org/contribute.html - in short:
+
+* Create an issue at http://projects.theforeman.org/projects/foreman/issues
+* Reference the issue via "Fixes #1234" or "Refs" in the commit message
+* Extract all strings for i18n, see Translation section in the guide
+* Prepend `[WIP]` or `WIP` to prevent lengthy tests for work in progress
```

Comment:
I'd suggest to always use ```[WIP]```. Keeps it simple even if the other option also works.